components/browser/engine-gecko/docs/metrics.md

```diff
@@ -22,9 +22,8 @@ In addition to those built-in metrics, the following metrics are added to the pi
 
 | Name | Type | Description | Data reviews | Extras | Expiration | [Data Sensitivity](https://wiki.mozilla.org/Firefox/Data_Collection) |
 | --- | --- | --- | --- | --- | --- | --- |
-| avif.aom_decode_error |[quantity](https://mozilla.github.io/glean/book/user/metrics/quantity.html) |Image-decode Error from AOM decoder  |[Bug 1670827](https://bugzilla.mozilla.org/show_bug.cgi?id=1670827#c9)|<ul><li>unit: error code</li></ul>|never | |
+| avif.aom_decode_error |[labeled_counter](https://mozilla.github.io/glean/book/user/metrics/labeled_counters.html) |Image-decode Error from AOM decoder  |[Bug 1690406](https://bugzilla.mozilla.org/show_bug.cgi?id=1690406#c3)|<ul><li>error</li><li>mem_error</li><li>abi_mismatch</li><li>incapable</li><li>unsup_bitstream</li><li>unsup_feature</li><li>corrupt_frame</li><li>invalid_param</li></ul>|never | |
```

Comment:
Should we be including these metrics changes in unrelated feature PRs? I've come across this before, but have I been doing it wrong?

Reply:
There was an update to Glean, which may have changed how the docs were generated, which is the only reason I included this one.